config/initializers/school.rb

```diff
@@ -0,0 +1,6 @@
+SCHOOLS = [
+  { claims: true },
+  { placements: true },
+  { claims: true, placements: true },
+  { claims: true, placements: true }
+].freeze
```

Comment:
I think this might be better located inside the `seeds.rb` file directly. From what I can tell, that's the only place this constant gets used?

I don't think this serves a clear purpose as a global constant, nor does it feel meaningful as a standalone file.

What do you think?